src/views/3-Modeling/Modeling.vue

```diff
@@ -4,7 +4,7 @@
       <h1 class="intro__hed">
         Modeling
       </h1>
-      <p> so much to say here!</p>
+      <p> so much to say here!!</p>
```

Comment:
LOL

Reply:
lol just needed to see if changes weren't just cached ;p